### PR TITLE
docs: update running on simulator iOS

### PR DESCRIPTION
**Pull request by luism3861**
this PR update iOS simulators according this PR https://github.com/react-native-community/cli/pull/1701

### CODE REVIEW

File: website/versioned_docs/version-0.70/running-on-simulator-ios.md [hunk 1]
```diff
@@ -9,13 +9,13 @@ Once you have your React Native project initialized, you can run `npx react-nati
 
 ## Specifying a device
 
-You can specify the device the simulator should run with the `--simulator` flag, followed by the device name as a string. The default is `"iPhone 13"`. If you wish to run your app on an iPhone SE (2nd generation), run `npx react-native run-ios --simulator='iPhone SE (2nd generation)'`.
+You can specify the device the simulator should run with the `--simulator` flag, followed by the device name as a string. The default is `"iPhone 14"`. If you wish to run your app on an iPhone SE (3rd generation), run `npx react-native run-ios --simulator='iPhone SE (3rd generation)'`.
```
Comment (cortinico) [inline website/versioned_docs/version-0.70/running-on-simulator-ios.md:12]: I'm unsure if we can update also the docs for 0.70.
The change seems to be on the CLI 9.1. Is that inside React Native 0.70?

Comment (luism3861) [reply website/versioned_docs/version-0.70/running-on-simulator-ios.md:12]: yep https://github.com/facebook/react-native/pull/34803

Comment (luism3861) [reply website/versioned_docs/version-0.70/running-on-simulator-ios.md:12]: Hi @cortinico thanks for the review, recently added see https://github.com/facebook/react-native/pull/34803

Comment (cortinico) [reply website/versioned_docs/version-0.70/running-on-simulator-ios.md:12]: Yup but this is on `main`, not on 0.70

**Review by cortinico — APPROVED** [review]

File: website/versioned_docs/version-0.70/running-on-simulator-ios.md (hunk 1 above)
Comment (cortinico) [reply website/versioned_docs/version-0.70/running-on-simulator-ios.md]: Yup it will go now in the `next` version, which will also become 0.71.0

**Review by Simek — APPROVED** [review]
As Nicola said, we can merge this in `next` now, thank you for the update! 🙏

**Pull request merged**